### PR TITLE
improved admin tabbing

### PR DESCRIPTION
**Pull request by stuartromanek**
## Summary
- increased focus visibility for admin bar piece manager buttons
- focus state for ‘subtle’ buttons (round + button in admin bar)
- improved accessibility / tab-ability in the locale switcher
- improved focus states for widget breadcrumbs
- context menus can be closed with `esc` .. focus returned to their parent trigger when closed
- All hidden area UI (area UI in its unhovered / unfocused state) now properly excludes itself from `tabindex` until hovered/focused (this was eating **a lot** of invisible focus tabbing)
- widget control UI reorganized for more intuitive tabbing order
- widget control UI focus state style improvements
- various accessibility tag updates

https://github.com/user-attachments/assets/6d9a3fef-a46c-4c08-ad0f-73a4d3553d3b

### CODE REVIEW

File: modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue [hunk 1]
```diff
@@ -50,6 +51,7 @@
               :tooltip="!isContextual && 'apostrophe:editWidgetForeignTooltip'"
               :icon-size="11"
               :modifiers="['no-motion']"
+              :disable-focus="!(isHovered || isFocused)"
```
Comment (ETLaurent) [inline modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue:54]: could we use `:tabbable="isHovered || isFocused"` here to be consistent?

**Review by ETLaurent — DISMISSED** [review]

**Review by ETLaurent — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,9 @@
 * Updates rich text link tool's keyboard key detection strategy.
 * Added tooltip for update button.
 
+### Adds
+ * Adds visual focus states and keyboard handlers for engaging with areas and widgets in-context
+
```
Comment (ETLaurent) [inline CHANGELOG.md:20]: duplicated `### Adds`

**Review by ETLaurent — CHANGES_REQUESTED** [review]

**Comment by ETLaurent** [comment]
image

focus on "image adjustement" and "close" icons seems broken, icons just shift a bit (no outline and nothing happens when pressing enter or space)

**Review by ETLaurent — APPROVED** [review]
Re-approving because I don't want to block the merge while I'm away.
The changelog just needs some format fixing

**Review by BoDonkey — APPROVED** [review]

**Review by BoDonkey — APPROVED** [review]

**Pull request merged**